pkg/client/local_intermediate_bundle/local_intermediate_bundle.go

```diff
@@ -0,0 +1,132 @@
+package intermediate
```

Comment:
this package is practically the same as the local one, just changing some filenames and extensions..

Reply:
why do we need a new configuration and a new package?